torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1126,6 +1126,12 @@ struct PythonPrintImpl {
         stmt << useOf(node->input(0)) << ".tolist()"
              << ")";
       } break;
+      case prim::EnumValue:
```

Comment:
Maybe add a comment here that these ops have to be resolved by the compiler instead of by schema matching for return type to be set appropriately

Reply:
Done.